projects/storefrontlib/src/lib/user/login/login.component.scss

```diff
@@ -1,101 +1,15 @@
 @import 'theme';
-@import 'functions';
-@import 'mixins';
-@import '~bootstrap/scss/nav';
-@import '~bootstrap/scss/dropdown';
 
-//login color variables
-$cx-login-status-greet-color: 'inverse' !default;
-$cx-login-status-md-greet-color: 'text' !default;
-$cx-login-status-toggle-color: 'inverse' !default;
-$cx-login-status-toggle-hover-color: 'primary' !default;
-$cx-login-status-md-toggle-color: 'text' !default;
-$cx-login-status-md-menu-background-color: 'transparent' !default;
-$cx-login-status-item-color: 'text' !default;
-$cx-login-status-item-hover-color: 'primary' !default;
-$cx-login-status-item-hover-background-color: 'transparent' !default;
+.cx-login-greet {
+  margin: var(--cx-margin, 0);
+  color: var(--cx-color, var(--cx-g-color-inverse));
+  @include type('7');
+  font-weight: var(--cx-font-weight, var(--cx-g-font-weight-bold));
```

Comment:
I know it used to be bold but it looks like it changed. If I look at the zeplin it looks more like a semi bold, maybe the old variable had a 600 value and not 700?

Reply:
fixed!